cluster_config.go

```diff
@@ -105,10 +106,16 @@ type Config struct {
 	// possible.
 	ReplicationFactorMin int
 
-	// MonitorPingInterval is frequency by which a cluster peer pings the
-	// monitoring component. The ping metric has a TTL set to the double
+	// MonitorPingInterval is the frequency by which a cluster peer pings
+	// the monitoring component. The ping metric has a TTL set to the double
 	// of this value.
 	MonitorPingInterval time.Duration
+
+	// PeerWatchInterval is the frequency that we watch for changes
```

Comment:
again minor typo but "frequency we use to watch for changes" or "frequency with which we watch for changes" is more correct.